test/Microsoft.AspNetCore.Mvc.Core.Test/RequireHttpsAttributeTests.cs

```diff
@@ -36,8 +41,8 @@ public void OnAuthorization_AllowsTheRequestIfItIsHttps()
                 // host, pathbase, path, query, expectedRedirectUrl
                 return new TheoryData<string, string, string, string, string>
                 {
-                    { "localhost", null, null, null, "https://localhost" },
-                    { "localhost:5000", null, null, null, "https://localhost:5000" },
+                    { "localhost", null, null, null, "https://localhost/" },
+                    { "localhost:5000", null, null, null, "https://localhost/" },
                     { "localhost", "/pathbase", null, null, "https://localhost/pathbase" },
                     { "localhost", "/pathbase", "/path", null, "https://localhost/pathbase/path" },
                     { "localhost", "/pathbase", "/path", "?foo=bar", "https://localhost/pathbase/path?foo=bar" },
```

Comment:
- Suggest adding an IPv6 example to this theory since the colons might confuse `UriBuilder` if `ToUriComponent()` isn't what it wants.
- Similar suggestion for adding a "本地主機:80" test.